edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/ShipmentLocationTO.java

```diff
@@ -13,21 +13,15 @@
 import java.time.OffsetDateTime;
 
 public record ShipmentLocationTO(
-  @Valid @NotNull
-  @RequireType(
-    value = {AddressLocationTO.class, UNLocationLocationTO.class, FacilityLocationTO.class},
-    message = "must be an AddressLocation, FacilityLocation or an UNLocationLocation"
-  )
-  LocationTO location,
-
-  @Size(max = 250)
-  String displayedName,
-
-  @NotNull
-  ShipmentLocationTypeCode shipmentLocationTypeCode,
-
-  OffsetDateTime eventDateTime
-) {
+    @Valid
```

Comment:
Did you actually do any changes here other than reformat to be less readable?

Reply:
Well, I use the google-java-format. Which is the one used for this repo.

Reply:
google-java-format doesn't tell you to remove blank lines.

Reply:
I reverted this change.